filter/date/filterdate.go

```diff
@@ -35,8 +34,10 @@ func DefaultFilterConfig() FilterConfig {
 				Type: ModuleName,
 			},
 		},
-		Format: []string{time.RFC3339Nano},
-		Source: "message",
+		Format:       []string{time.RFC3339Nano},
+		Source:       "message",
+		Target:       "@timestamp",
+		TagOnFailure: []string{"gogstash_filter_date_error", "_dateparsefailure"},
```

Comment:
Why TagOnFailure "_dateparsefailure" as a default value?
I mean there was "gogstash_filter_date_error"

Reply:
That was specifically to ensure some sort of compatibility with the output that logstash's date plugin dumps into elasticsearch, so I had added it to the list of tags added on failure to not break compatibility with anything anyone had been doing.

Reply:
I think it's better to remove from default value, because not all users migrate from logstash, and it's unnecessary for pure gogstash user.
Write some examples or hints in README for your case?